app/uk/gov/hmrc/cgtpropertydisposals/repositories/enrolments/TaxEnrolmentRepository.scala

```diff
@@ -50,7 +51,15 @@ class DefaultTaxEnrolmentRepository @Inject() (mongo: MongoComponent)(implicit
       mongoComponent = mongo,
       collectionName = "tax-enrolment-requests",
       domainFormat = TaxEnrolmentRequest.format,
-      indexes = Seq(IndexModel(ascending("ggCredId"), IndexOptions().name("ggCredIdIndex"))),
+      indexes = Seq(
+        IndexModel(
+          ascending("ggCredId"),
+          IndexOptions()
+            .name("ggCredIdIndex")
+            .expireAfter(24, TimeUnit.HOURS)
```

Comment:
Can we make these configurable via config? So that we are not hardcoding the expiry period

Reply:
@mahesh-hmrc Made changes according to the comment.